integration/cursors/awaitdata_test.go

```diff
@@ -308,6 +308,122 @@ func TestCursorsTailableAwaitData(t *testing.T) {
 	})
 }
 
+func TestCursorsTailableAwaitDataAfterInsertStress(t *testing.T) {
+	if !setup.IsMongoDB(t) {
+		t.Skip("https://github.com/FerretDB/FerretDB/pull/3961")
+	}
+
+	var count atomic.Int32
+
+	teststress.Stress(t, func(ready chan<- struct{}, start <-chan struct{}) {
+		testID := count.Add(1)
+
+		t.Run(fmt.Sprint(testID), func(t *testing.T) {
```

Comment:
`t.Run` doesn't run in parallel, effectively un-stressing the test

Reply:
From my testing it seems it's not

Reply:
Ok, please explain how `t.Run` works then